app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find_by(id: params[:followed_id])
```

Comment:
bỏ () ko cần thiết em hè
```suggestion
    @user = User.find_by id: params[:followed_id]
```